csss-site/src/events/frosh/views/frosh2023.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import render, redirect
+from random import randrange
+from django.http import Http404
+
+from events.frosh.views.create_frosh_context import create_frosh_context
+
+
+def index(request):
+    return render(request, 'frosh/2023/index.html', create_frosh_context())
+
+
+def frosh(request):
+    return render(request, 'frosh/2023/frosh.html', create_frosh_context())
+
+
+def secret(request):
+    test = randrange(0, 10)
+    if test < 7:
+        raise Http404('Not sure if there is a secret here. Please refresh your page and try again.')
+    return redirect('https://www.youtube.com/watch?v=dQw4w9WgXcQ', permanent=True)
```

Comment:
i don't see the point to using `redirect` here over `HttpResponseRedirect`? and if i am not mistaken, using `redirect` [might actually be more unnecessary work in this case](https://stackoverflow.com/a/13305294)?

Reply:
I copied this from last year. All I know is that the code works

Reply:
> Also, make sure not to prematurely optimize

my man, you argued that this can be done via a github page, I doubt you're gonna be utilizing the additional options of `redirect` if that's the case 😂

Reply:
> I directly copied this from last year

things can slip by me, I ain't perfect 🙃

Reply:
please switch to `HttpResponseRedirect`. If you don't know how to. I can do it for you.

Reply:
I would prefer to leave it as `redirect` because it is clearer to read. Since this is likely only going to be run a few times ever, readability is more important than performance, in this case. As the stackoverflow you linked claimed, `redirect` is a more flexible alternative to `HttpResponseRedirect`.

Reply:
ordinarly, I would not care, but in this case, a precedent has already been set and the SO is actually a mark against you imo cause you're using something with more features than you need.

Reply:
> I also like how redirect is shorter. So I'd use redirect over HttpResponseRedirect.
> 
> Both are fine to use though.

This is a quote from the SO. The only precedent I see being set is that one _should_ use `redirect` instead of `HttpResponseRedirect`, which is what's being done right here.